.github/workflows/bc_check.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Run Bridgecrew
+on:
+  pull_request:
+    branches: [master, feature/cbl/deployment-config]
```

Comment:
Should we support that branch `feature/cbl/deployment-config`?

Reply:
Since this is a new test that is not currently on the main branch, this needs to be present for running the actions on this branch, I'm going to remove it before merging.